src/erlcloud_aws.hrl

```diff
@@ -0,0 +1,199 @@
+-ifndef(erlcloud_aws_hrl).
```

Comment:
Let's remove this vendored file or document exactly what problem we were hitting that led us to add it. Further if we do add it here, we'll need to include the license.

Reply:
File deleted.
https://buildkite.com/chef-oss/chef-mini-s3-master-verify/builds/5
https://buildkite.com/chef/chef-chef-server-master-verify/builds/2425#16139667-3e0b-4d14-b425-a855b4f86e48
https://buildkite.com/chef/chef-chef-server-master-omnibus-adhoc/builds/1659#1e73e43d-30bc-4cad-ac2f-f8627dcd5fb8